docs/getting-started/detections-req.asciidoc

```diff
@@ -35,6 +35,65 @@ of at least 32 characters. For example:
 IMPORTANT: After changing the `xpack.encryptedSavedObjects.encryptionKey` value
 and restarting {kib}, you must restart all detection rules.
 
+[discrete]
+[[enable-and-access-detections]]
+== Enable and Access Detections
+Enabling the Detections feature allows you to view and use the Detections page from within the Security app. You only need to complete this action once and can choose between enabling it for a single Kibana space or across multiple ones. Once the detections feature is enabled, additional privileges are needed to use Detection rules and alerts, described in the following table.
```

Comment:
> You only need to complete this action once and can _choose between enabling it for a single Kibana space or across multiple ones._

Not sure I'm following the distinction here?  This action will need to be completed on each space they intend to enable detections, as there is no way to bulk setup multiple spaces via the UI (can be automated via the API though). Does 'choose between enabling...' mean here to "use this role and go to each space to complete setup"?